src/Web/MeAjudaAi.Web.Admin/README.md

```
@@ -2,6 +2,20 @@
 
 Portal administrativo Blazor WebAssembly para gerenciamento da plataforma MeAjudaAi.
 
+## 📑 Índice
+
+- [Quick Start](#quick-start)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)